Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -85,7 +85,7 @@
       <SubType>Code</SubType>
     </Compile>
     <Compile Include="$(MSBuildThisFileDirectory)Issue9006.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8207.xaml.cs" >
+    <Compile Include="$(MSBuildThisFileDirectory)Issue8207.xaml.cs">
```

Comment:
Please revert all these whitespace changes so we can reduce conflicts.

Reply:
Reverted.